app/forms/respondents_detail.rb

```diff
@@ -14,7 +14,7 @@ class RespondentsDetail < BaseForm
   attribute :email_address, :string
   attribute :fax_number, :string
   attribute :organisation_employ_gb, :integer
-  attribute :organisation_site_number, :integer
+  attribute :organisation_site_number, :boolean
```

Comment:
I cant help thinking this is a bad name for a boolean attribute ?  What do you think ?

Reply:
Updated in 44c2ceb and then fixed merge conflicts in 8048c6d.

Thanks for pointing this out, you're right, it was a bad variable name.